packages/Overview.mdx

```diff
@@ -25,111 +25,87 @@ som kan brukes til å videreutvikle og lage mer avanserte og sammensatte kompone
     marginBottom: 'var(--ds-spacing-12)',
   }}
 >
-  <Card
-    isLink
-    className='sb-unstyled'
-    asChild
-  >
-    <a
-      href='https://github.com/digdir/designsystemet/blob/main/README.md'
-      target='_blank'
-      rel='noreferrer'
+  <Card className='sb-unstyled'>
+    <Heading
+      level={2}
+      size='md'
       className='sb-unstyled'
+      asChild
     >
-      <Card.Header className='sb-unstyled'>
-        <Heading
-          level={2}
-          size='md'
+      <h2 className='sb-unstyled'>
+        <a
+          href='https://github.com/digdir/designsystemet/blob/main/README.md'
+          target='_blank'
+          rel='noreferrer'
           className='sb-unstyled'
-          asChild
         >
-          <h2 className='sb-unstyled'>Kom i gang</h2>
-        </Heading>
-      </Card.Header>
-      <Card.Content className='sb-unstyled'>
-        Les vår README-fil for å komme i gang med å bruke designsystemet.
-      </Card.Content>
-    </a>
+          Kom i gang
+        </a>
+      </h2>
```

Comment:
This card now has a `p` inside of an `h2`.
![image](https://github.com/user-attachments/assets/cdcf200d-94fb-4d39-9f9c-589dd96dd270)
![image](https://github.com/user-attachments/assets/f3c0d468-2d89-4f18-87dc-de92223dbbe2)

This is likely due to storybook trying to parse that line as a string, and adding it as a paragraph.

Reply:
Oh bra sett! Fikset det :)